src/Ydb.Sdk/src/Services/Table/Session.cs

```diff
@@ -60,9 +35,9 @@ protected virtual void Dispose(bool disposing)
         {
             if (_sessionPool is null)
             {
-                _logger.LogTrace($"Closing detached session on dispose: {Id}");
+                Logger.LogTrace($"Closing detached session on dispose: {Id}");
 
-                var client = new TableClient(Driver, new NoPool());
+                var client = new TableClient(Driver, new NoPool<Session>());
                 _ = client.DeleteSession(Id, new DeleteSessionSettings
```

Comment:
Why do you need TableClient for delete session?
What about move the method to session pool?

(may be in other pr)

Reply:
old code